simulation-debugging.md

```diff
@@ -75,3 +65,57 @@ libsystem_kernel.dylib`__read_nocancel:
 ```
 
 The lldb or gdb shells behave like normal sessions, please refer to the LLDB / GDB documentation.
+
+The last parameter, the &lt;viewer\_model\_debugger&gt; triplet, is actually passed to make in the build directory, so
+
+```sh
+make posix_sitl_lpe jmavsim___gdb
+```
+
+is equivalent with
+
+```sh
+make posix_sitl_lpe	# Configure with cmake
+make -C build_posix_sitl_lpe jmavsim___gdb
+```
+
+A full list of the available make targets in the build directory can
+be obtained with:
+
+```sh
+make help
+```
+
+but for your convenience, a list with just the &lt;viewer\_model\_debugger&gt; triplets
+is printed with the command
+
+```sh
+make list_vmd_make_targets
```

Comment:
This needs to be changed to `make posix list_vmd_make_targets`

@bkueng Are there any values than "posix" above that work?